### DISCUSSION TITLE
LISEM run data error

### DISCUSSION THREAD

**Discussion by lozaayehu** [bastianvandenbout/LISEM#11] [Q&A]
Dear Prof,
I would like to share with you the error I encountered while running my data for a Simple slope stability analysis. I constantly have this error message on the Console box. Could you please guide me through this? I copy and past the error, I constantly have  Thank you so much for your help.

"compiling C:/Program Files/lisem/bin/kernels/model_flow.cl flow
compilation error clBuildProgram(õ) CL_BUILD_PROGRAM_FAILURE
Build log for gfx902: C:\Users\acer\AppData\Local\Temp\\OCL17332T1.cl:247:37: error: read_image expects readable image argument float rain = read_imagef(RAIN,sampler, int2(gx_x1,gy)).x; ~~~~~~~~~~~^ 1 error generated. error: Clang front-end compilation failed! Frontend phase failed compilation. Error: Compiling CL to IR 
CL ERROR, QUITTING MODEL!
Deleting all model-related memory objects
Finished deleting all model related memory objects
Do Model Run on thread"

**Comment by bastianvandenbout** [comment]
Dear Loza, 

I see, this can happen sometimes when you run the model on the GPU. It typically means the complex gpu code for the model is not supported.
Could you try to run it using CPU (there is in the main settings a 'use cpu" option at the bottom)

If you have a dedicated GPU, it might be good to use that instead. In nvidia/amd settings, you can indicate if you want a software to start with the intel integrated graphics or with the dedicated graphics card.

**Comment by lozaayehu** [comment]
Okay prof, I also understood that I need to minimize my raster data. Happen to that, I used GIS to clip my raster data. However, the clip file is not accurate when it comes to LISEM. Therefore,  is there any option available to clip raster data to a smaller size (rectangle polygon) on LISEM software
Best regards, [cid:86fa2747-3706-4f47-95cb-0b126c58fb05]
________________________________
